README.md

```diff
@@ -75,7 +75,7 @@ Parameter | Description | Required | Default
     <plugin>
         <groupId>io.swagger</groupId>
         <artifactId>swaggerhub-maven-plugin</artifactId>
-        <version>1.0.2</version>
+        <version>1.0.3-SNAPSHOT</version>
```

Comment:
you want the docs to point to the current dev version as opposed to the latest release version?